projects/demo-maps/src/app/route-components/route-example-cesium/route-example-cesium.component.ts

```diff
@@ -197,12 +199,43 @@ export class RouteExampleCesiumComponent implements OnInit, OnDestroy {
       }),
       new VectorLayer({
         id: 'geojson_test',
-        name: 'GeoJSON Vector Layer',
+        name: 'GeoJSON Vector Layer (default)',
         attribution: `© DLR GeoJSON`,
         type: 'geojson',
         data: testData,
         visible: false,
         popup: true
+      }),
+      new VectorLayer({
+        id: 'geojson_test2',
+        name: 'GeoJSON Vector Layer (styled)',
+        attribution: `© DLR GeoJSON`,
+        type: 'geojson',
+        data: testData,
+        visible: false,
+        popup: true,
+        options: {
+          style: (feature: Feature) => {
+            let styles = [];
+
+            let polygonStyle = new Style({
+              stroke: new Stroke({
+                color: '#0072a3',
+                width: 1
+              }),
+              fill: new Fill({
+                color: '#FFFFFF' + '99',
+
+              }),
+            });
+            styles.push(polygonStyle);
+            return styles;
+          },
+          fillColor: '#FFFFFF',
+          strokeColor: '#0072a3',
+          strokeWidth: 2,
+          clampToGround: false
```

Comment:
Maybe we can find a better solution here so that the values are not duplicated in the object.

Here is how this object is used in the other mapping libraries

**map-ol:**
- https://github.com/dlr-eoc/ukis-frontend-libraries/blob/dd51dd64632f13d483ee0405618abf0f911c0c03/projects/map-ol/src/lib/map-ol.service.ts#L1140
- https://github.com/dlr-eoc/ukis-frontend-libraries/blob/dd51dd64632f13d483ee0405618abf0f911c0c03/projects/map-ol/src/lib/map-ol.service.ts#L1184
- https://github.com/dlr-eoc/ukis-frontend-libraries/blob/dd51dd64632f13d483ee0405618abf0f911c0c03/projects/map-ol/src/lib/map-ol.service.ts#L933

**map-maplibre:**
- https://github.com/dlr-eoc/ukis-frontend-libraries/blob/dd51dd64632f13d483ee0405618abf0f911c0c03/projects/map-maplibre/src/lib/maplibre-layers.helpers.ts#L188

Reply:
I found a way to access the style properties in a similar way as in map-maplibre.  Therefore, I removed the duplicate entries.

Reply:
nice.